lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -1262,6 +1291,7 @@ def ensemble_diffusion(self, method, *, max_lag=None):
         if method == "cve":
             return ensemble_cve(self)
```

Comment:
I think that while this one will work, we should probably remove the localization uncertainty estimate for "mixed" image acquisition `Kymo`s, since these will depend on the `Kymo` in question. The risk is otherwise that the user gets one number and proceeds to plug this into the per-track estimations.

Reply:
👍